src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds.IntegrationTests/RefundsTestContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.ProviderPayments.Calc.Refunds.Infrastructure.Entities;
+using SFA.DAS.ProviderPayments.Calc.Refunds.IntegrationTests.Attributes;
+
+
+namespace SFA.DAS.ProviderPayments.Calc.Refunds.IntegrationTests
+{
+    /// <summary>
+    /// Used for suite-wide data
+    /// </summary>
+    static class RefundsTestContext
+    {
+        /// <summary>
+        /// Populated by applying the <see cref="SetupUkprnAttribute"/> to the test
+        /// </summary>
+        public static long Ukprn { get; set; }
+
+        /// <summary>
+        /// Populated by applying the <see cref="SetupPaymentsHistoryAttribute"/> to the test
+        /// </summary>
+        public static List<HistoricalPaymentEntity> PaymentsHistory { get; set; }
+    }
```

Comment:
I always worry about static values, this may be problematic when tests run concurrently. I understand that NUnit does support a Context, perhaps we can use this (not a major issue)